frontend/src/components/ui/Table.tsx

```diff
@@ -1,32 +1,176 @@
+          {name.split(' ')[1]}
+        </span>
+      </td>
+    );
+  }
+  if (name === DashboardColumns.FUNCTION) {
+    return (
+      <td className="flex flex-col">
+        <span className="block">
+          {name.split(' ')[0]} {name.split(' ')[1]}
+        </span>
+        <span className="block">{name.split(' ')[2]}</span>
+      </td>
+    );
+  }
+  if (name === DashboardColumns.RESPECTFUL) {
+    return (
+      <td className="flex flex-row  flex-nowrap text-nowrap ">
+        {name.split(' ')[0]} <br />
+        {name.split(' ')[1]} {name.split(' ')[2]}
+      </td>
+    )
+  }
+    if (name === DashboardColumns.WILLINGNESS) {
+      return (
+        <td className="flex flex-row  flex-nowrap text-nowrap ">
+          {name.split(' ')[0]} <br />
+          {name.split(' ')[1]}
+        </td>
+      );
+  } else return <td className="flex flex-row  flex-nowrap text-nowrap ">{name}</td>;
+};
 export const Table = ({
   pageParams,
   handlePageParams,
   handleChangeRowsPerPage,
   tableData,
+  columns,
+  onChangeTab,
+  tabs,
+  selectedTab,
+  counts,
+  role,
 }: {
   pageParams: PageParams;
   handlePageParams: (data: Partial<DashboardFilters>) => void;
   handleChangeRowsPerPage: (data: Partial<DashboardFilters>) => void;
   tableData: TableData;
+  columns: DashboardColumns[];
+  onChangeTab: (index: number) => void;
+  tabs: TabData[];
+  selectedTab: number;
+  counts: TabCount;
+  role?: Role;
 }) => {
   const { rows, totalRows } = tableData;
 
   return (
-    <table className="border border-separate border-spacing-0 order-slate-400 shadow-lg rounded-md w-full bg-white ">
-      <TableHeader columns={tableData.columns} />
+    <table className="table-auto overflow-x-scroll w-auto border-separate border border-slate-100 z-40">
+      <thead>
+        <tr>
+          <th
+            colSpan={columns.length}
+            className="mt-2 align-left sticky   top-[80px] bg-white w-full h-[80px]"
+          >
+            <div className="w-full text-left py-8  caption-top bg-white">
+              <h4 className="font-bold px-4">Search Results</h4>
+            </div>
+          </th>
+        </tr>
+
+        {role === Role.COORDINATOR && (
+          <tr>
+            <th
+              colSpan={columns.length}
+              className="bg-white border-b border-b-separate border-b-slate-100 align-left w-full sticky top-[180px] h-[45px]"
+            >
+              <Tabs
+                onChangeTab={onChangeTab}
+                tabs={tabs}
+                selectedTab={selectedTab}
+                counts={counts}
+              />
+            </th>
+          </tr>
+        )}
+        <tr>
+          <th
+            className={`sticky h-[10px] bg-white top-[226px] border-b-1 border-slate-100`}
+            colSpan={columns.length}
+          ></th>
+        </tr>
+        <tr
+          className={`sticky px-6  bg-white  h-[48px] shadow-blue-gray-900 ${role === Role.COORDINATOR ? ' top-[225px]' : 'top-40'}`}
```

Comment:
Historically, I've seen tailwind have issues with conditional classNames like this, which is why I've had to define two different classNames in their own strings, and use the defined strings within the conditional. Are you not seeing issues here?

Reply:
am not having any issues - I think before it might have been b/c we were passing in the value, rather than setting it conditionally inline? (just a guess)